tasks/build_btrust.yml

```diff
@@ -0,0 +1,41 @@
+---
+# This is included in deploy.yml only if {{ a2dp_sink_btrust_binary_url }} is not defined
+
+- name: Install toolchain
+  shell: curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
Is there no PPA or something similar for this?

Reply:
Some distros package `rustup`, but not debian. See https://github.com/rust-lang/rustup.rs/issues/800